modules/test/playwright/tests/portal-web/html/taglib/ui/roleSelector.spec.ts

```diff
@@ -5,17 +5,43 @@
 
 import {expect, mergeTests} from '@playwright/test';
 
+import {dataApiHelpersTest} from '../../../../../fixtures/dataApiHelpersTest';
 import {loginTest} from '../../../../../fixtures/loginTest';
 import {PORTLET_URLS} from '../../../../../utils/portletUrls';
 import {waitForAlert} from '../../../../../utils/waitForAlert';
 
-const test = mergeTests(loginTest());
+const test = mergeTests(dataApiHelpersTest, loginTest());
 
 test(
 	'Multiple selection with pagination in SearchContainer modal',
 	{tag: '@LPD-50672'},
-	async ({page}) => {
+	async ({apiHelpers, page}) => {
+		const maxNumberOfRoles = 15;
 		let iframeSelectRole;
+		const roleIds = [];
+
+		const companyId = await page.evaluate(() => {
+			return Liferay.ThemeDisplay.getCompanyId();
+		});
+
+		await test.step('Create extra roles', async () => {
+			for (let i = 0; i <= maxNumberOfRoles; i++) {
```

Comment:
This will make 16 roles, while max number of roles is 15 :slightly_smiling_face: